03_Collection Types.playground/section-1.swift

```diff
@@ -18,6 +18,7 @@ import UIKit
 
 // Arrays and dictionaries in swift use generics and can be mutable or immutable
 // depending on whether they are assigned to a var or let
+// they are VALUES type and a STRUCT, so must be assigned to new variable when using non mutating internal function
```

Comment:
Yes, because in Obj-C usually people work directly in array so it have some difference with this, it's struct mean can assign to another var or let without needing to  use `copy` or bother with mutable array, It's basically the same thing as what you have stated in the Class and Struct playground, non mutating function is `reverse` and some `high order functions`